app/src/main/java/sopt/org/millie/presentation/search/SearchScreen.kt

```diff
@@ -0,0 +1,181 @@
+package sopt.org.millie.presentation.search
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import sopt.org.millie.R
+import sopt.org.millie.core.designsystem.component.MillieSearchTextField
+import sopt.org.millie.core.designsystem.component.MillieTabbar
+import sopt.org.millie.core.designsystem.component.MillieTopappbar
+import sopt.org.millie.core.designsystem.theme.MillieTheme
+import sopt.org.millie.presentation.search.book.SearchBookScreen
+import sopt.org.millie.presentation.search.library.SearchLibraryScreen
+import sopt.org.millie.presentation.search.model.SearchBannerModel
+import sopt.org.millie.presentation.search.model.SearchBookModel
+import sopt.org.millie.presentation.search.model.SearchLibraryModel
+
+@Composable
+fun SearchScreen(
+    searchBookList: List<SearchBookModel>,
+    searchLibraryList: List<SearchLibraryModel>,
+    searchBanner: SearchBannerModel,
+    onClickBookItem: () -> Unit,
```

Comment:
```suggestion
    onBookItemClick: () -> Unit,
```
보통 Click을 뒤로 보내주는게 컴포즈 컨벤션이에요 ! 그리고 해당 콜백은 단순 클릭이 아니라 선택된 책의 id를 넘겨주는 역할이라서 () -> Unit보다는 (Long) -> Unit 형태로 받아주는 게 타입 안정성 측면에서 더 명확할 것 같아용 ~~